tests/integration/pilot/vm/README.md

```diff
@@ -0,0 +1,79 @@
+# Writing Tests for VMs
+
+Support for including VMs in the integration tests has been added to the framework.
+This document describes the integration and extension mechanisms to exercise VM related code.
+The primary goals are to test VM-related code & ensure it works on a range of supported OS types and versions.
```

Comment:
Can we list out the goals below instead of putting them in one paragraph?

Reply:
Updated.